csrankings.csv

```diff
@@ -16153,3 +16153,4 @@ François Fleuret,EPFL,https://people.epfl.ch/francois.fleuret,Bj1tRlsAAAAJ
 Alexandre Alahi,EPFL,https://people.epfl.ch/alexandre.alahi,UIhXQ64AAAAJ
 Pierre Vandergheynst,EPFL,https://people.epfl.ch/pierre.vandergheynst,1p9NOFEAAAAJ
 Ali H. Sayed,EPFL,https://people.epfl.ch/ali.sayed,uDG9sXQAAAAJ
+Xiaojun Chen,Shenzhen University, http://www.escience.cn/people/chenxiaojun/index.html,yAjyrwkAAAAJ

```

Comment:
Remove spurious space after comma.

Please confirm with a link to an appropriate page that this person is part of the CS department or otherwise meets the inclusion criteria.